### DISCUSSION TITLE
Is there any API that allows us to extract actual monthly cost of a database or organization?

### DISCUSSION THREAD

**Discussion by fshuman-a** [planetscale/discussion#631] [Q&A] [answered]
I noticed there is an [API](https://api-docs.planetscale.com/reference/get_organizations_name) to get organization info but I only see **expected monthly budget**. Instead, I'm looking for the **actual monthly cost**. Is there an API or webhook for that?
Thanks!

**Comment by mscoutermarsh** [comment]
Will get back to you on this soon. We do have an API for this, I need to check on the permissions for service tokens.

As an alternative, cost monitoring is also available via Vantage: https://planetscale.com/docs/integrations/vantage.

**Reply by fshuman-a** [reply]
Ah thanks. We use a different service to see the costs (CloudZero) and wanted to grab the PlanetScale costs to post them there.

**Reply by mscoutermarsh** [reply, marked as answer]
Docs for this are being published later. You can grab invoice information by adding the `read_invoices` permission to your service token.

**Get all invoices:**
`api.planetscale.com/v1/organizations/org-name/invoices`

This will get you the current total cost for the month.

**Invoice line items:**

This will get you a break down of how much per database.
`api.planetscale.com/v1/organizations/org-name/invoices/id/line-items`

The data for these are updated once per hour.

If you're logged in as an admin in your org, you can also hit these in your browser. Your cookie will let you access the info.

**Reply by mscoutermarsh** [reply]
Update: [Billing data now available via the API](https://planetscale.com/changelog#billing-data-now-available-via-the-api)

**Reply by fshuman-a** [reply]
Thank you @mscoutermarsh. I can now see the invoices 👍
